test/functional/test-xhr.js

```diff
@@ -413,4 +413,49 @@ describe('XHR', function() {
 
     });
   });
+
+  describe('FetchResponse', () => {
+    const mockXhr = {
+      status: 200,
+      responseText: 'this is some test text',
+    };
+
+    it('should provide text', () => {
+      const response = new FetchResponse(mockXhr);
+      return response.text().then(result => {
+        expect(result).to.equal('this is some test text');
```

Comment:
Well, I was avoiding that in case FetchResponse modified it by side effect accidentally or something.  I moved both strings out to a testText var that I reference both places.